<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>526</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>251</width>
     <height>313</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="7" column="0">
     <widget class="QPushButton" name="BtnSimpan">
      <property name="text">
       <string>SIMPAN</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QPushButton" name="BtnUbah">
      <property name="text">
       <string>UBAH</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QPushButton" name="BtnHapus">
      <property name="text">
       <string>HAPUS</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="idPesanLabel">
      <property name="text">
       <string>Id Pesan</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="idPesanLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="idMemberLabel">
      <property name="text">
       <string>Id Member</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="idMemberLineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="idMobilLabel">
      <property name="text">
       <string>Id Mobil</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="idMobilLineEdit"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="tawarLabel">
      <property name="text">
       <string>Tawar</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="tawarLineEdit"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="tanggalLabel">
      <property name="text">
       <string>Tanggal</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QDateEdit" name="tanggalDateEdit"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="pesanLabel">
      <property name="text">
       <string>Pesan</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="pesanLineEdit"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="statusLabel">
      <property name="text">
       <string>Status</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="statusLineEdit"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
